kong/init.lua

```diff
@@ -956,7 +956,7 @@ function Kong.init_worker()
   end
 
   if is_not_control_plane then
-    plugin_servers.start()
+    assert(plugin_servers.start())
```

Comment:
Is it a good idea to fail a worker starting? For some of the deployments it will cause the workers to be restarted repeatly.